Sources/Plasma/FeatureLib/pfMetalPipeline/plMetalPipelineState.h

```diff
@@ -50,6 +50,10 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "plMetalDevice.h"
 #include "plSurface/plShaderTable.h"
 
+#ifndef PLASMA_FORCE_PER_PIXEL_LIGHTING
+#define PLASMA_FORCE_PER_PIXEL_LIGHTING 0
+#endif
```

Comment:
Is this define used anywhere now, or left over from the earlier version where it was a compile-time decision?

Reply:
oh, I see this is used in plMetalPipeline.cpp... maybe it makes sense to move the definition there?

Reply:
This is used to force per pixel lighting as always on for all surfaces, which is handy for debugging.

Reply:
I've moved the preprocessor define into the plMetalPipeline header. Although now I'm wondering if it should go into the cpp...

Reply:
Ok, it's moved to the cpp.